news/2020-02-04-community-contributors-2019.md

```diff
@@ -0,0 +1,212 @@
+
+### osu!catch
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Jemzuu.jpg)](https://osu.ppy.sh/users/7890134)
+
+After a bit of a rocky start, osu!catch's longest standing nominator in terms of consecutive time (430 days!) has been nothing less than extraordinary over the last year. Boasting nearly unmatched statistics in resets, disqualifications, and nominations on top of an incredibly impressive tenure in the Quality Assurance Helpers, **Jemzuu** has been a massive help in just about every field that osu!catch has to offer. When it comes to being an Elite Nominator, it would be nearly impossible to overlook him.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/wonjae.jpg)](https://osu.ppy.sh/users/5032045)
+
+Clocking in at just over a year served in the Beatmap Nominators, **wonjae**'s consistently high nomination activity has always been a massive boon to the group. In times when things have been slow, wonjae's strong presence on the nomination front cannot be understated. Leading the nomination numbers nearly every month is no small feat, especially on top of the generally low amount of resets he gets in comparison. It's for these reasons that wonjae joins the prestigious group of Elite Nominators.
+
+### osu!mania
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/MysticEyes.jpg)](https://osu.ppy.sh/users/6253266)
+
+**-MysticEyes** left a strong impression during his year in the Beatmap Nominators. Being the most active osu!mania BN, he also rarely had any issues in their nominations despite having the highest activity of all osu!mania nominators. On top of this, he was not scared to push maps different from what usually would get ranked. -MysticEyes would even be extremely kind and helpful to everyone - in every way, he really was the dream nominator!
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Kobii.jpg)](https://osu.ppy.sh/users/6209713)
+
+One of the most active osu!mania nominators since joining in early 2019, **\_Kobii** has been active in pushing maps in not just the 4k mode, or the 7k mode, but also 6k, 9k, and more! On top of that, they have been very approachable for mappers new and old in the osu!mania mapping community and serve as a great example of what a Beatmap Nominator can be!
+
+-------
+
+2019 was one of the strongest years osu! has had so far, and it is in no small part due to the efforts of those listed above.
+
+We know that so many people contribute to help make osu! awesome, and those we have awarded today are by no means the sole, exhaustive list of everyone who deserves one.
+
+osu! is unique among the thousands of communities on the internet in that we are shaped so very closely by those among us who choose to help, and we are all privileged to share in the result of so many people working together for a common goal.
```

Comment:
unsure, but shouldnt this be 'amongst' ?

Reply:
US vs UK english, amongst is preferred in UK, among in US